index.html

```diff
@@ -4,9 +4,70 @@
     <meta charset="UTF-8">
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>Weather Report</title>
+    <title > Min's Weather Report</title>
+    <link href = styles/index.css rel = "stylesheet">
 </head>
 <body>
     
+    <main >
+        <header>
+            <h1 id = "weather-title">  Min's Weather Report </h1>
+            <span>
+                For the lovely city of
+                <span id= "CityName" class = "headerCityName">Miami</span>
+
+            </span>
+        </header>
+    </main>
+    <section class = " Big-Container " >
+
+            <div class="Temp-Container">
+                
+                <div class =Temp-Content>
+                    
+                    <h1>Temperature</h1>
+                    <span id="tempvalue" class="plum">50</span>
+                    <div id = tempButtons>
+                        <button id= "increaseButton">+</button>
+                        <button id = "decreaseButton">-</button>
+                    </div>
+                
+                </div>
+            </div>
+                
+            <div class="Sky-Container">
+                <h1>Sky</h1>
+                <select class = "skyDropMenu">
+                    <option value = sunny >Sunny</option>
+                    <option value = cloudy>Cloudy</option>
+                    <option value = rainy>Rainy</option>
+                    <option value =snowy>Snowy</option>
+                </select>
+            </div>
+            <div class="City-Container">
+                <h1>City</h1>
+                <div id = cityNameAndBtn>
+                    <input type ="text" id ="cityNameInput" value = "Miami">
+                    <input type = "reset"  value = "Reset"></input>
+                </div>
+            </div> 
+
+    </section >
+        
+    <section class="smallContainer"> 
+            
+            <div class= "gardenContainer "> 
+                <h2 id=gardenText></h2>
+                
+            </div>
+                        <br> 
+            <div class = "landscape">🥵🔥🔥🔥🔥🔥🔥🔥🔥🔥🔥🔥🔥🔥🔥🔥🔥🔥🔥🔥🥵</div>
+    </section>
+    <script async src="src/index.js"></script>
 </body>
-</html>
\ No newline at end of file
+</html>
+
+<!-- ‼️❗️🤯😳☀️😎 🌆🌇🎇🥵💃🏾💋‼️❗️🤯 -->
+<!-- 🥶🧊💎🥶🧊💎🥶🧊💎🥶🧊💎🥶🧊💎🥶🧊💎🥶🧊💎🥶🧊💎🥶🧊💎🥶
+✈️☁️✌🏾✈️☁️✌🏾✈️☁️✌🏾✈️☁️✌🏾✈️☁️✌🏾✈️☁️✌🏾✈️☁️✌🏾✈️☁️✌🏾✈️☁️✌🏾✈️☁️✌🏾✈️☁️✌🏾✈️☁️✌🏾✈️☁️
+🌧☔️🌂🌧☔️🌂🌧☔️🌂🌧☔️🌂🌧☔️🌂🌧☔️🌂🌧☔️🌂🌧☔️🌂🌧☔️🌂🌧☔️🌂🌧☔️🌂 -->
```

Comment:
I would remove these emojis in your HTML file because you are using them in your JS file.